app/experimenter/kinto/tasks.py

```diff
@@ -215,7 +247,9 @@ def nimbus_check_experiments_are_live():
                 experiment.publish_status = NimbusExperiment.PublishStatus.IDLE
                 experiment.save()
 
-                generate_nimbus_changelog(experiment, get_kinto_user())
+                generate_nimbus_changelog(
+                    experiment, get_kinto_user(), message="Experiment is now live!"
```

Comment:
What are your thoughts on making changelog messages constant keys and letting the frontend decide which message to display? Or even storing this and the `Timed Out` messages as constants in the backend? Would love to see these messages grouped/managed together

Reply:
Yeah I filed another ticket to dig into this more deeply https://github.com/mozilla/experimenter/issues/5022 you're right we should store them as constants.